ixwebsocket-config.cmake.in

```diff
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if（@USE_ZLIB@）
```

Comment:
if（@USE_ZLIB@）

@bsergean 
I'm very sorry, I just found out that the brackets in the above sentence are not in the right format, they are not English characters. So could you please help to fix it, thank you very much!

Reply:
Made a different issue to fix this -> https://github.com/machinezone/IXWebSocket/issues/387